packages/graphql/src/schema/make-augmented-schema.test.ts

```diff
@@ -104,24 +104,6 @@ describe("makeAugmentedSchema", () => {
         expect(() => makeAugmentedSchema({ typeDefs })).toThrow("cannot have interface on relationship");
     });
 
-    test("should throw type X does not implement interface X correctly", () => {
```

Comment:
We should probably leave a negative test for interface implementation in, perhaps moved into validate-document.test.ts instead.

Reply:
There is one https://github.com/neo4j/graphql/pull/232/files#diff-1332c2df9bfa483fbae8bd4af0e5b04092e34ad090756e1b895845c04e2d4c2dR192

Reply:
I mean there should be a test to ensure that an error _is_ thrown if a type doesn't implement an interface correctly.

Reply:
Since we have removed our own custom logic to do with the matter I don't think it is necessary to test. The problem is now outside our domain and shouldn't be our concern.

Reply:
I think that given we're using a non-default export from the GraphQL library, we should have a few common positive and negative test cases just to prove functionality. Lot's of good "should not throw" tests here, but not as many "should throw" tests (without which, we have little confidence that the function will not always "not throw").